torch/testing/_internal/two_tensor.py

```diff
@@ -36,6 +37,8 @@ def __new__(cls, a, b, outer_size=None, outer_stride=None):
         assert a.storage_offset() == b.storage_offset()
         return out
 
+    @torch._disable_dynamo
+    @_mark_subclass_constructor_exportable_experimental
```

Comment:
tbh this API is already in the private `torch._export` namespace, maybe we should just remove this leading underscore + experimental part of the string (otherwise you're going to have to change it at some point anyway, and if people start using it you'll still need to support both versions)